packages/cerebro/LICENSE.md

```
@@ -27,6 +27,425 @@ The published @visulima/cerebro artifact additionally contains code with the fol
 MIT
 
 # Bundled dependencies:
+## @visulima/command-line-args
+License: MIT
+By: Daniel Bannert
+Repository: git+https://github.com/visulima/visulima.git
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)